src/components/ComputeModal.js

```diff
@@ -101,6 +93,34 @@ const DataprocDiskSelector = ({ value, onChange }) => {
   ])
 }
 
+const SparkInterface = ({ interfaceName, synopsis, namespace, name, onDismiss }) => {
+  const interfaceDisplayName = Utils.switchCase(interfaceName,
+    ['yarn', () => 'YARN Resource Manager'],
+    ['apphistory', () => 'YARN Application Timeline'],
+    ['hdfs', () => 'HDFS NameNode'],
+    ['sparkhistory', () => 'Spark History Server'],
+    ['jobhistory', () => 'MapReduce History Server']
+  )
+
+  return div({ style: { ...styles.whiteBoxContainer, marginBottom: '1rem', backgroundColor: colors.accent(0.1), boxShadow: Style.standardShadow } }, [
+    div({ style: { flex: '1', lineHeight: '1.5rem', minWidth: 0, display: 'flex' } }, [
+      div([
```

Comment:
Can it be replaced with `h(Fragment`? It's not apparent if the div is actually providing style, but since it only has children and no style attribute, it doesn't seem to be adding anything valuable to the DOM on the surface.

Reply:
Yea I thought the same and tried `h(Fragment` but it looked like below

![image](https://user-images.githubusercontent.com/5438223/135648337-7e8b45ac-ff79-4ff1-b005-55bca6d88b1c.png)